iree/hal/api.cc

```diff
@@ -1080,8 +1080,12 @@ IREE_API_EXPORT iree_status_t IREE_API_CALL iree_hal_device_queue_submit(
   }
 
   // For now we always go to the first compute queue. TBD cleanup pending the
```

Comment:
nit: outdated comment - no longer always going to the first compute queue after the simple `queue_affinity` handling.